docs/developers-clear-containers-install.md

```diff
@@ -0,0 +1,141 @@
+# Developers Clear Containers 3.0 Install
+
+This guide is not intended for end-users. Instead, this guide provides
+instructions for any developers eager to try Clear Containers 3.0 and who
+want to build Clear Containers from the source code and are familiar with the
+process. Basic programming knowledge and familiarity with the Linux command
+line are expected.
+
+## Requirements
+
+  * [go 1.8.3](https://golang.org/)
+  * [glibc-static](https://www.gnu.org/software/libc/libc.html) Required to build **Virtcontainers pause**
```

Comment:
For consistency (see `(pause binary only)` below), I'd put the text at the end of this line in brackets.

Also, does "Virtcontainers pause" have to be in bold? It's not a warning and I don't think that element is any more "special" than the other parts, so I'd personally just remove the bold formatting.